CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 ### Added
 
 - Add support for `DNSEndpoint` CRs. See README for further information.
+- Add support for Google Cloud `gcp` provider.
```

Comment:
This should be moved to [Unreleased]. Guess this happened during merging master into your branch.